packages/kbn-dev-utils/src/ci_stats_reporter/ci_stats_reporter.ts

```diff
@@ -90,20 +90,24 @@ export class CiStatsReporter {
       this.log.debug(e.message);
     }
 
+    const isElasticCommitter = email && email.endsWith('@elastic.co') ? true : false;
+
     const defaultMetadata = {
-      osPlatform: Os.platform(),
-      osRelease: Os.release(),
-      osArch: Os.arch(),
+      committerHash: email
+        ? crypto.createHash('sha256').update(email).digest('hex').substring(0, 20)
+        : undefined,
       cpuCount: Os.cpus()?.length,
       cpuModel: Os.cpus()[0]?.model,
       cpuSpeed: Os.cpus()[0]?.speed,
+      email: isElasticCommitter ? email : undefined,
       freeMem: Os.freemem(),
-      totalMem: Os.totalmem(),
-      committerHash: email
-        ? crypto.createHash('sha256').update(email).digest('hex').substring(0, 20)
-        : undefined,
-      isElasticCommitter: email ? email.endsWith('@elastic.co') : undefined,
+      isElasticCommitter,
       kibanaUuid,
+      nestedTiming: process.env.CI_STATS_NESTED_TIMING,
```

Comment:
We support booleans in metadata, we should use one here

Reply:
f407e5be155f205376f2b5c0afe0ce00a36ee9d9